contracts/Unipool.sol

```diff
@@ -135,8 +132,25 @@ contract Unipool is LPTokenWrapper {
         lastUpdateTime = block.timestamp;
         periodFinish = block.timestamp.add(DURATION);
 
-        tradedToken.safeTransferFrom(msg.sender, address(this), _amount);
+        rewardToken.safeTransferFrom(msg.sender, address(this), _amount);
```

Comment:
Should we add a `require`?

Reply:
No need, it does it in the safeErc20 library, in fact this function doesn't return anything.

Reply:
Ah yeah i see, this is a different version from the one used by Aragon right?